app/layout.tsx

```diff
@@ -22,7 +23,7 @@ export default function RootLayout({
         <Header />
         <div className="main-layout">
           <Navbar />
-          <div className="container">{children}</div>
+          <div className="pl-[50px] lg:pl-[200px] w-full">{children}</div>
```

Comment:
pls fix the screen 836px
![image](https://github.com/linhdatran3/nextjs-14/assets/72385907/b5735a8a-e73f-4b7d-921e-6aacdcb15d83)

Reply:
Added responsive support for medium width screen